FFXIVClientStructs/FFXIV/Client/UI/UIModule.cs

```diff
@@ -219,6 +231,9 @@ public static void PlayChatSoundEffect(uint effectId)
     [MemberFunction("E8 ?? ?? ?? ?? 4D 39 BE")]
     public static partial bool PlaySound(uint effectId, long a2, long a3, byte a4);
 
+    [MemberFunction("E8 ?? ?? ?? ?? 84 C0 74 BF 4C 8B CB")]
+    public static partial bool IsPlayerCharacterName(byte* name);
```

Comment:
Add `[GenerateCStrOverloads]` to generate string overloads automatically

Reply:
Updated